mix.lock

```diff
@@ -1,7 +1,7 @@
 %{"certifi": {:hex, :certifi, "0.3.0"},
   "earmark": {:hex, :earmark, "0.1.19"},
   "ex_doc": {:hex, :ex_doc, "0.11.2"},
-  "excoveralls": {:git, "https://github.com/parroty/excoveralls.git", "a0d3d57b7137f3204690e4126a82d48d985a09fb", [tag: "v0.4.3"]},
+  "excoveralls": {:git, "https://github.com/parroty/excoveralls.git", "a0d3d57b7137f3204690e4126a82d48d985a09fb", [tag: "v0.4.5"]},
   "exjsx": {:hex, :exjsx, "3.2.0"},
   "exlager": {:git, "https://github.com/khia/exlager.git", "abb13435a540731b8b9c5f649f4d94f11a27c1f7", []},
```

Comment:
Should exlager be removed here, too? `mix deps.clean --unused` should do the trick.